pallets/elections/src/migrations/v0_to_v1.rs

```diff
@@ -73,3 +73,60 @@ pub fn migrate<T: Config, P: PalletInfoAccess>() -> Weight {
     StorageVersion::new(1).put::<P>();
     T::DbWeight::get().reads(reads) + T::DbWeight::get().writes(writes)
 }
+
+#[cfg(feature = "try-runtime")]
+pub fn pre_upgrade<T: Config, P: PalletInfoAccess>() -> Result<(), &'static str> {
+    match Members::<T>::get() {
+        Some(_) => {}
+        _ => return Err("No `Members` storage"),
+    }
+
+    if StorageVersion::get::<P>() == StorageVersion::new(0) {
+        Ok(())
+    } else {
+        Err("Bad storage version")
+    }
+}
+
+#[allow(dead_code)]
+#[cfg(feature = "try-runtime")]
+pub fn post_upgrade<T: Config, P: PalletInfoAccess>() -> Result<(), &'static str> {
+    let mps = match MembersPerSession::get() {
+        Some(mps) => mps,
+        _ => return Err("No `MembersPerSession` in the storage"),
+    };
+    let reserved_members = match ReservedMembers::<T>::get() {
+        Some(rm) => rm,
+        _ => return Err("No `ReservedMembers` in the storage"),
+    };
+    let non_reserved_members = match ReservedMembers::<T>::get() {
```

Comment:
Change request: that's strange, because you take here `ReservedMembers` (by accident I guess), which then is successfully checked against `.is_empty` - this should fail, shouldn't it?

Reply:
fixed